addon/index.js

```diff
@@ -359,6 +361,15 @@ function installExperiment(experiment) {
   }
 }
 
+function uninstallSelf() {
+  // First, kick out all the experiment add-ons
+  Object.keys(store.installedAddons).forEach(id => {
+    uninstallExperiment({addon_id: id});
+  });
```

Comment:
I just renamed `uninstall-all` to `uninstall-self` in the README, since it turns out that's basically what I did :)